### ISSUE TITLE
Idk if you are updating this but here's my suggestion

### ISSUE THREAD

**Issue by TeteuXD2** [Rdimo/Hazard-Nuker#189]
Nuke a server that the user owns (or admin), this will need the server id

**Comment by dekrypted** [comment]
So like add a server nuker? Or add an option which nukes servers that the owner is in?

If so, I'm currently working on a server nuker so yeah.

**Comment by IchNukeDichWeg** [comment]
> > So u mean if bot is in a server but ur not and u nuke with guild id?
> 
> Yes!
> You just need the guild ID (As long as the account you're nuking is in said server & has admin permissions)
> 
> I can make it list all servers and what permissions you have in them

Yes ik I was asking the other guy if it was that what he ment or not.
